File: javascript/raindrops/raindrops.js
export const convert = (num) => {
  const sounds = {
    3: 'Pling',
    5: 'Plang',
    7: 'Plong'
  }
  return Object.entries(sounds).map(sound => {
    if (num % +sound[0] == 0) {
      return sound[1]
    }
  }).join('') || num.toString();
};

File: python/raindrops/raindrops.py
#def convert(number):
#    result = ""
#    if (number % 3 == 0):
#        result = result + "Pling"
#    if (number % 5 == 0):
#        result = result + "Plang"
#    if (number % 7 == 0):
#        result = result + "Plong"
#    if result == "":
#        return str(number)
#    else:
#        return result
#
def convert(number):
    return ''.join({ key:value for (key,value) in 
        {3:"Pling", 5:"Plang", 7:"Plong"}.items()
        if number % key == 0 }.values()) or str(number)

